helix-core/src/main/java/org/apache/helix/task/TaskRebalancer.java

```diff
@@ -219,6 +227,32 @@ public ResourceAssignment computeBestPossiblePartitionState(ClusterDataCache clu
     return newAssignment;
   }
 
+  private Set<String> getWorkflowAssignedInstances(String currentJobName,
```

Comment:
Method name should indicate that the returned value does not consider the current job.